test/shopify_app/session/session_repository_test.rb

```diff
@@ -94,15 +87,32 @@ class SessionRepositoryTest < ActiveSupport::TestCase
     test '.store_user_session stores a user session' do
       SessionRepository.user_storage = InMemoryUserSessionStore
 
-      session = ShopifyAPI::Session.new(
-        domain: 'shop.myshopify.com',
-        token: 'abracadabra',
-        api_version: :unstable
-      )
+      session = mock_shopify_session
       user = { 'id' => 'abra' }
       InMemoryUserSessionStore.expects(:store).with(session, user)
 
       SessionRepository.store_user_session(session, user)
     end
+
+    private
+
+    def mock_shopify_domain
+      'shop.myshopify.com'
+    end
+
+    def mock_shopify_user_id
+      'abra'
+    end
+
+    def mock_shopify_session
+      ShopifyAPI::Session.new(
+        domain: mock_shopify_domain,
+        token: 'abracadabra',
+        api_version: :unstable,
+        extra: {
+          scopes: 'read_products',
+        }
```

Comment:
Is this still used anywhere in this file?

Reply:
Yeah I just forgot to update this. Good catch!